### PR TITLE
feat: (CXSPA-8041) - Fix status as 'Selected' is not conveyed to the user in CheckoutDeliveryAddressComponent

### PR DESCRIPTION
**Pull request by petarmarkov9449**
Closes: [CXSPA-8041](https://jira.tools.sap/browse/CXSPA-8041)

Fix status as 'Selected' is not conveyed to the user in CheckoutDeliveryAddressComponent

### CODE REVIEW

**Review by Pio-Bar — DISMISSED** [review]

File: projects/storefrontlib/shared/components/card/card.component.html [hunk 1]
```diff
@@ -3,10 +3,13 @@
   class="cx-card"
   [tabindex]="border ? 0 : -1"
   [attr.role]="content.role ?? null"
-  [attr.aria-label]="
-    content.label ? (content.label | cxTranslate: { number: this.index }) : null
+  [attr.aria-labelledby]="
+    content.header ? 'content-header' + (index >= 0 ? '-' + index : '') : null
+  "
+  [attr.aria-describedby]="
```
Comment (Zeyber) [inline projects/storefrontlib/shared/components/card/card.component.html]: Because we already have the `ariaDescribedBy()` getter, could we have this logic in there? It might also be good to have `ariaLabelledBy()` to do the same with. We should add comments and tests too as the logic is conditional.

**Review by Zeyber — CHANGES_REQUESTED** [review]

File: projects/storefrontlib/shared/components/card/card.component.html (hunk 1 above)
Comment (petarmarkov9449) [reply projects/storefrontlib/shared/components/card/card.component.html]: I've updated it.

File: projects/storefrontlib/shared/components/card/card.component.html [hunk 2]
```diff
@@ -25,11 +23,18 @@
       {{ content.deleteMsg }}
     </div>
     <!-- Card title -->
-    <span *ngIf="content.title" class="cx-card-title">
+    <span
+      *ngIf="content.title"
+      class="cx-card-title"
+      [id]="'content-title' + (index >= 0 ? '-' + index : '')"
```
Comment (Zeyber) [inline projects/storefrontlib/shared/components/card/card.component.html]: Could we please use a get method for these `span`s too?

**Review by Zeyber — CHANGES_REQUESTED** [review]

File: projects/storefrontlib/shared/components/card/card.component.html (hunk 2 above)
Comment (petarmarkov9449) [reply projects/storefrontlib/shared/components/card/card.component.html]: I've just updated it.

File: projects/storefrontlib/shared/components/card/card.component.html [hunk 3]
```diff
@@ -3,9 +3,7 @@
   class="cx-card"
   [tabindex]="border ? 0 : -1"
   [attr.role]="content.role ?? null"
-  [attr.aria-label]="
-    content.label ? (content.label | cxTranslate: { number: this.index }) : null
-  "
+  [attr.aria-labelledby]="ariaLabelledBy"
```
Comment (Zeyber) [inline projects/storefrontlib/shared/components/card/card.component.html]: Do we still need `ariaLabelledBy` if we have `getLabel`?

**Review by Zeyber — CHANGES_REQUESTED** [review]

File: projects/storefrontlib/shared/components/card/card.component.html (hunk 3 above)
Comment (petarmarkov9449) [reply projects/storefrontlib/shared/components/card/card.component.html]: I've just updated it.

**Review by Zeyber — DISMISSED** [review]

**Comment by developpeurweb** [comment]
@petarmarkov9449 this should get narrated in a similar way as they do in My Account section, please reach out to @uroslates for feedback on how he fixed a similar bug in My Account.
The original bug for the ticket is now fixed, however a new bug was introduced, the narration is now repeated twice when focusing on the card and also while focusing on the button, as per screenshot below. 
While you are it, please exclude the button from the card narration (as in My Account for Address and Payment).

**In short:**
- When focusing the card, everything should be narrated only once, excluding the button.
- When focusing the button, only the button should be narrated.

payment method repetition

shipping address repetition

**Review by developpeurweb — APPROVED** [review]
